program/src/instructions/compressed/buy.rs

```diff
@@ -151,10 +153,12 @@ pub fn process_buy<'info>(
     let list_state = &ctx.accounts.list_state;
 
     // In case we have an extra remaining account.
+    // TODO: no checked math?
```

Comment:
good call but in this case it's never going to overflow as remaining_accounts length is limited by solana runtime and transaction limitations